.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "extern/sila_java"]
 	path = extern/sila_java
 	url = https://gitlab.com/SiLA2/sila_java.git
+	branch = usc
```

Comment:
@FlorianBauer remove this line, it should then work

Reply:
The reference to sila_java is correct (110ec60ea9a49f3a00a1b0a49c9654365ec7025f) but it seems that the CI does not pull the right version of sila_java